labs/src/main/scala/org/scalalabs/basic/lab01/HelloWorldExercise.scala

```diff
@@ -45,7 +45,7 @@ object HelloWorld {
    * More on method declarations can be found here:
    * http://programming-scala.labs.oreilly.com/ch02.html#MethodDeclarationsAndDefinitions
    */
-	def echo(text: String): String = "FixMe"
+	def echo(text: String): String = "Echo"
```

Comment:
I see that this passed tests, but I'm not sure if it matches the intent of a function with signature `echo(text: String)`